Public/Src/Utilities/Utilities/VmCommandProxy/VmTypes.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+
+namespace BuildXL.Utilities

```

Comment:
>BuildXL.Utilities [](start = 10, length = 17)

Currently, these are the only contract we have between us and CB for interaction with VmCommandProxy